regulations/static/regulations/js/source/views/main/star-processor.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+var $ = require('jquery');
+
+/***
```

Comment:
Not sure if we're trying to use jsdoc-style comments, but I figure we might as well. By convention, they look like this:

```javascript
/**
 * comment
 */
```